tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38
+envlist = py39
```

Comment:
unrelated (and unused in CI) change, just matching python version required elsewhere.